### ISSUE TITLE
->unsigned() missing

### ISSUE THREAD

**Issue by renege** [laracasts/Laravel-5-Generators-Extended#78]
When adding a relation with the generator, it generates for example:

```
            $table->integer('user_id');
            $table->foreign('user_id')->references('id')->on('users');
```

But then `->unsigned();` is missing on the integer itself.

**Comment by scalerboy** [comment]
Add :unsigned. 

eg. user_id:integer:unsigned:foreign

**Comment by alameya** [comment]
it should be putted automaticly because it always present in primary key 'id'

**Comment by Maidomax** [comment]
+1

**Comment by philsown** [comment]
BTW, "unsigned" is also the 3rd argument to integer. You can do:

`$table->integer('user_id', false, true);`

**Comment by tabacitu** [comment]
Thanks everyone for pitching in. This project didn't get the attention it deserves until now - we're working on v2 which will include a fix for this. I've already merged #165 into the ```v2``` branch, which fixes this afaik. Let me know if it doesn't.

Cheers!
